go/vm/lfvm/instructions.go

```diff
@@ -677,6 +677,20 @@ func opBaseFee(c *context) {
 	}
 }
 
+func opBlobHash(c *context) {
```

Comment:
```suggestion
if c.isCancun() {
	c.status = INVALID_INSTRUCTION
	return
}

index := c.stack.pop()
lenght :_= uint64(len(c.context.GetTransactionContext().BlobHashes)
if index.IsUint64() && index.Uint64() < lenght) {
     stack.push(c.context.GetTransactionContext().BlobHashes[index.Uint64()][:])
} else{
   stack.push(0)
}
```
this is completely optional, but the point here is:

1. early bail out, avoid nesting ifs
2. smaller boolean expressions, no call/compute in boolean code
3. explicit stack operations that reflect instruction spec.
4. positive boolean, instead of ! + || (some linters would ask for parenthesis here)

Reply:
plz. check replacement, there are some bugs. github does not allow me to edit it anymore.